packages/patternfly-4/gatsby-ssr.js

```diff
@@ -16,6 +16,6 @@ exports.onRenderBody = ({ setHtmlAttributes, setPostBodyComponents, setHeadCompo
   ]);
   // QUALTRICS WEBSITE FEEDBACK SNIPPET
   setHeadComponents([
-      <script type="text/javascript" src="qualtrics.js"/>
+      <script type="text/javascript" src="/surveys/qualtrics.js" />
```

Comment:
Rather than adding this as a component to the `<head>`, can we bundle it by using `import './qualtrics.js';` in `packages/patternfly-4/gatsby-browser.js`?

Reply:
The purpose of adding it to the head is so no matter what part of the sight you access the survey will appear. I thought doing it this way was so that I did not have to edit multiple pages. Am i wrong about this approach? I was just follow the Gatsby docs.